subsystem/framework/framework.go

```diff
@@ -243,32 +211,30 @@ func (r *TerraformRunner) Run(args ...string) int {
 	GinkgoWriter.Println(&outb)
 	GinkgoWriter.Println(&errb)
 
-	r.LastRunOutput = RunOutput{
-		out: outb.String(),
-		err: errb.String(),
+	return RunOutput{
+		out:        outb.String(),
+		err:        errb.String(),
+		StatusCode: cmd.ProcessState.ExitCode(),
```

Comment:
a np one
can we change the attribute name `StatusCode` to `ExitCode`? Current one easily to mislead others to regard it as the status code from http response

Reply:
Oh I forgot to do that